shawerma/models.py

```diff
@@ -0,0 +1,35 @@
+import datetime
```

Comment:
please add `from __future__ import unicode_literals` on top - this will avoid having exceptions when you add some special characters

https://stackoverflow.com/questions/23370025/what-is-unicode-literals-used-for